components/src/Select/Select.story.js

```diff
@@ -142,10 +161,10 @@ storiesOf("Select", module)
       helpText="Additional information about input"
     />
   ))
-  .add("With smaller maxHeight", () => (
+  .add("with smaller maxHeight", () => (
     <Select
       initialIsOpen
-      maxHeight="128px"
+      maxHeight="132px"
```

Comment:
What's this change for?

Reply:
To show a better example of scrolling affordance. It now cuts off in the middle of a word, instead of near the top, making it very obvious that there is scrollable content in the box.